cmd/lotus-miner/sectors.go

```diff
@@ -1515,6 +1516,192 @@ var sectorsUpdateCmd = &cli.Command{
 	},
 }
 
+var sectorsExpiredCmd = &cli.Command{
+	Name:  "expired",
+	Usage: "Get or cleanup expired sectors",
+	Flags: []cli.Flag{
+		&cli.BoolFlag{
+			Name:  "show-removed",
+			Usage: "show removed sectors",
+		},
+		&cli.BoolFlag{
+			Name:  "remove-expired",
+			Usage: "remove expired sectors",
+		},
+
+		&cli.Int64Flag{
+			Name:   "confirm-remove-count",
+			Hidden: true,
+		},
+		&cli.Int64Flag{
+			Name:        "expired-epoch",
+			Usage:       "epoch at which to check sector expirations",
+			DefaultText: "WinningPoSt lookback epoch",
+		},
+	},
+	Action: func(cctx *cli.Context) error {
+		nodeApi, closer, err := lcli.GetStorageMinerAPI(cctx)
+		if err != nil {
+			return err
+		}
+		defer closer()
+
+		fullApi, nCloser, err := lcli.GetFullNodeAPI(cctx)
+		if err != nil {
+			return xerrors.Errorf("getting fullnode api: %w", err)
+		}
+		defer nCloser()
+		ctx := lcli.ReqContext(cctx)
+
+		maddr, err := nodeApi.ActorAddress(ctx)
+		if err != nil {
+			return xerrors.Errorf("getting actor address: %w", err)
+		}
+
+		// toCheck is a working bitfield which will only contain terminated sectors
+		toCheck := bitfield.New()
+		{
```

Comment:
Just for my education what's the point of having code blocks like this?

Reply:
Generally just for not leaking variables into the wider function context (which makes it easier to see when we are done with those vars)